client/mydonor/src/app/customer/customer-feedback/customer-feedback.component.ts

```diff
@@ -18,6 +18,7 @@ export class CustomerFeedbackComponent {
     let id = this.token.getDecodedToken().nameidentifier;
     this.service.feedbackReg(this.model, id).subscribe({
       next: (Data) => {
+        console.log(Data);
```

Comment:
Remove console.log statements. Also fix the naming convention violation of variable Data.